main.py

```diff
@@ -29,7 +29,7 @@ def parse_message(message):
         if 'dnf' in message.lower().split()[0]:
             return float('inf')
         message = message.replace(":", "")
-        return int(message)
+        return int(message.split()[0])
     except (ValueError, AttributeError) as e:
```

Comment:
```suggestion
    except (ValueError, AttributeError, IndexError) as e:
```
Need to catch IndexError for the removed message bug